proto/src/determined/modelmetrics/v1/modelmetrics.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package determined.modelmetrics.v1;
+option go_package = "github.com/determined-ai/determined/proto/pkg/modelmetricsv1";
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+// The current state of the metrics.
+enum State {
+  // The state of the metrics is unknown.
```

Comment:
Semantically, this is all kinda funny, I agree with your earlier points. It's weird for metrics to have state. Maybe these comments should be worded like `The state of the workload associated with some batch of metrics...`.

Reply:
Doesn't make any difference if you just change a comment here. : ) And we are removing workloads. So it doesn't make sense to mention workloads either.